enhancements/installer/packet-ipi.md

```diff
@@ -0,0 +1,313 @@
+  currently requires a minimum of 100GB per volume. Additionally, the [Packet
+  EBS service](https://www.packet.com/developers/docs/storage/ebs/) is available
+  in a limiting subset of the available facilities.
+- Both Public and Private IPv4 and IPv6 addresses made available for each node.
+  Private and VLAN Networks can be restricted at the project level (default) or
+  node by node. While custom ASNs and BGP routing features are supported, these
+  and other networking features will not be exposed through the IPI.
+  <https://www.packet.com/cloud/network/>
+- Openshift will run on RHCOS directly on bare metal without the need for a
+  VM layer.
+
+### Risks and Mitigations
+
+The [official Packet
+ClusterAPI](https://github.com/kubernetes-sigs/cluster-api-provider-packet),
+[CCM](https://github.com/packethost/packet-ccm), and
+[CSI](https://github.com/packethost/csi-packet) drivers are all relatively new
+and may make undertake substantial design changes.  This project will either
+need to adopt and maintain the current releases, or adopt newer releases
+developed alongside this project.
+
+The CI will need to assure that Packet resources are cleanly released after
+failed and successful builds.  Packet is currently used in the bare metal
+provisioning tests, so there is some prior art available.  If the same account
+is used for testing the IPI, it may need account quota increases.
+
+## Design Details
+
+### Open Questions
+
+- How will the API be LoadBalanced?
+  - Will this be a task left up to the user (and added documentation)?
+  - Will extra devices be dedicated to this purpose?
+  - Will DNS round-robin suffice?
+  - Can Kube-vip be used for this purpose?
+  - Can the API be loadbalanced by the service Loadbalancer?
+    [openshift/enhancements#459](https://github.com/openshift/enhancements/pull/459)
+- Is the Packet CCM necessary?
+  - One of the Packet CCM functions is to provider LoadBalancer support through a (CCM) managed deployment of MetalLB. Enabling LB support is optional.
+    - If enabled, who is responsible for supporting the MetalLB installation?
+    - If not enabled, how do we take advantage of [Service Load Balancers for On
+      Premise
+      Infrastructure](https://github.com/openshift/enhancements/pull/356) for
+      use with Packet (and Packet BGP features)?
+  - Can [https://github.com/plunder-app/kube-vip#56](plunder-app/kube-vip)
+    stand-in for LoadBalancer service on Packet while disabling the packet-ccm's
+    MetalLB approach to LoadBalancing?
+  - Aside from loadbalancing, are other CCM responsibilities critical? These
+    include Kubernetes node annotations and Packet API hints that a "device"
+    (and therefor node) has been deleted.
+- Is the Packet CSI necessary? If not, how will the Openshift image registry be
+  created.
+- Can the sig-cluster-lifecycle ClusterAPI and OpenShift MachineAPI converge to
```

Comment:
I don't think this is likely to happen any time soon. The models are very very different at this point sadly. I would expect we can reuse a lot of the upstream provider pretty verbatim but the source of configuration will have to remain embedded in the Machine for the foreseeable future which will be the major difference between the two

Reply:
@JoelSpeed Do you think I should drop the question or leave it as an open area of concern or interest?

Reply:
Might be worth including my comments as a reply? That or drop it, either way works for me

Reply:
Added.